client/models.py

```diff
@@ -0,0 +1,51 @@
+from django.db import models
+from django.core.validators import MinLengthValidator
+from django.core.exceptions import ValidationError
+from buy_together_app.models import User
+
+
+class Client(User):
+    area = models.CharField(max_length=32,
+                            validators=[MinLengthValidator(1)],
+                            help_text="area")
+
+    @staticmethod
+    def get_all_clients():
+        """Fetches list of all Clients.
+        Args:
+            None.
+        Returns:
+            returns a queryset containing all the Clients.
+        """
+        return Client.objects.all()
```

Comment:
Is there a need to wrap the Django builtin method? if not, this method is redundant and can be removed. 
We should not maintain code the Django team already maintain.